utils.py

```diff
@@ -16,6 +16,9 @@
 import shutil
 import yaml
 
+import numpy as np
+import nibabel as nib
```

Comment:
It is not included in `requirements.txt`

Reply:
Fixed in https://github.com/spinalcordtoolbox/manual-correction/pull/15/commits/06dba920ec84786573596de77cd4da943b1c8e8f